packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -358,6 +375,24 @@
         }
     ],
     "metrics": [
+        {
+            "name": "languageServer_setup",
```

Comment:
can we port this to the commons repo? I think we wanted to align all teams on a common implementation